frontend/views/pages/Page.vue

```diff
@@ -85,29 +91,46 @@ $pagePaddingDesktop: 75px;
 
 .p-main {
   grid-area: p-main;
-  overflow: auto;
+  padding-left: $spacer;
+  padding-right: $spacer;
+  margin: 0 auto;
+  max-width: 43rem;
+  @include overflow-touch;
+
+  &.full-width {
+    max-width: 100%;
+  }
+
+  @include tablet {
+    padding-left: $pagePaddingTablet;
+  }
+
+  @include widescreen {
+    margin-left: 0;
+    padding-left: $pagePaddingDesktop;
```

Comment:
Is it suppose to have this space so large? In the designs, it seems the cards to up to the edge of the main, with smaller padding.

Reply:
yes it's supposed to be like this from what I can see